reloader_plugin.py

```diff
@@ -106,14 +106,14 @@ def theBestShortcutForPluginReload(self):
   def initGui(self):
     self.actionRun = QAction(
       QIcon(os.path.join(os.path.dirname(__file__), "reload.png")),
-      u"Reload chosen plugin",
+      "Reload chosen plugin",
       self.iface.mainWindow()
     )
-    self.actionRun.setWhatsThis(u"Reload chosen plugin")
+    self.actionRun.setToolTip("Reload chosen plugin")
     plugin = currentPlugin()
     if plugin:
-      self.actionRun.setWhatsThis(u"Reload plugin: %s" % plugin)
-      self.actionRun.setText(u"Reload plugin: %s" % plugin)
+      self.actionRun.setToolTip("Reload plugin: {}".format(plugin))
```

Comment:
Do you know when the good old formatting is going to be deprecated?

Reply:
No idea, sorry...

Reply:
Thank you! :*